composer.json

```diff
@@ -101,5 +101,11 @@
             "allow-contrib": false,
             "require": "4.4.*"
         }
-    }
+    },
+    "funding": [
+        {
+            "type": "paypal",
+            "url": "https://paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=ETT4JRJARLTSC"
+        }
+    ]
 }
```

Comment:
already on master. How come we apply the change again?

Reply:
I'll rebase

Reply:
done